esx_service/cli/vmdkops_admin.py

```diff
@@ -118,7 +118,8 @@ def commands():
                 '-c': {
                     'help': 'Display selected columns',
                     'choices': ['volume', 'datastore', 'created-by', 'created',
-                                'attached-to', 'policy', 'capacity', 'used'],
+                                'attached-to', 'policy', 'capacity', 'used',
+                                'access', 'attach-as'],
```

Comment:
I plan to change the column name in CLI output to "access" and "attach-as". Otherwise, it will need a lot of changes in the code. There is no extra advantage to name those two columns as "access-mode" and "attach-mode".